### DISCUSSION TITLE
Decimal type not allowed in paths methods

### DISCUSSION THREAD

**Discussion by nikita-gorodeckij** [StellarCN/py-stellar-base#594] [Ideas]
https://github.com/StellarCN/py-stellar-base/blob/5c97fa0bb87ce1160719353934bdf1f6a301af99/stellar_sdk/server.py#L252
https://github.com/StellarCN/py-stellar-base/blob/5c97fa0bb87ce1160719353934bdf1f6a301af99/stellar_sdk/server.py#L272

I think here should be allowed Decimal as well as in other places. You allow Decimal in transaction builder but here not, I think it is strange.

**Comment by overcat** [comment]
Thanks for your feedback, added in [8.0.1](https://pypi.org/project/stellar-sdk/8.0.1/).
